### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by TheBodger**
> Please send your pull requests the develop branch.
> Don't forget to add the change to CHANGELOG.md.

**Note**: Sometimes the development moves very fast. It is highly
recommended that you update your branch of `develop` before creating a
pull request to send us your changes. This makes everyone's lives
easier (including yours) and helps us out on the development team.
Thanks!

* Does the pull request solve a **related** issue?
* If so, can you reference the issue?
* What does the pull request accomplish? Use a list if needed.
added a new script entry to the json.package that can be used in a windows OS environement to start MM2 using electron more easily. 
* If it includes major visual changes please add screenshots.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "js/electron.js",
   "scripts": {
     "start": "DISPLAY=\"${DISPLAY:=:0}\" ./node_modules/.bin/electron js/electron.js",
+    "windows": "./node_modules/.bin/electron js/electron.js",
```
Comment (rejas) [inline package.json:8]: maybe rename it to start:windows ?

**Comment by MichMich** [comment]
Not sure what this PR accomplishes.

**Comment by rejas** [comment]
I think he wants to start the project under windows :-)

**Comment by MichMich** [comment]
`npm run start` should work. We don't need to make a modification for this.

**Comment by rejas** [comment]
The windows command line doesnt like the ```DISPLAY="${DISPLAY:=:0}"````  at least on my win10 pc.

**Comment by TheBodger** [comment]
Hi




That is correct. I am looking to add support for  MM2 running natively under Windows. The start command with display doesn't work under Windows.




Sent from Outlook Mobile







On Thu, Apr 30, 2020 at 9:07 AM +0100, "Veeck" <notifications@github.com> wrote:















The windows command line doesnt like the ```DISPLAY="${DISPLAY:=:0}"````  at least on my win10 pc.

**Comment by MichMich** [comment]
We don't officially support windows, so unfortunately I'm not going to implement this. 
If you want you can always start your project by directly calling `./node_modules/.bin/electron js/electron.js`.

Thanks.

**Comment by TheBodger** [comment]
Thanks for taking the time to respond and consider it.

**Pull request closed without merging**